src/main/java/net/snowflake/client/jdbc/DefaultResultStreamProvider.java

```diff
@@ -94,20 +97,19 @@ private HttpResponse getResultChunk(ChunkDownloadContext context) throws Excepti
 
     if (context.getChunkHeadersMap() != null && context.getChunkHeadersMap().size() != 0) {
       for (Map.Entry<String, String> entry : context.getChunkHeadersMap().entrySet()) {
-        SnowflakeResultSetSerializableV1.logger.debug(
-            "Adding header key={}, value={}", entry.getKey(), entry.getValue());
+        logger.debug("Adding header key={}, value={}", entry.getKey(), entry.getValue());
```

Comment:
Are we sure none of these headers are sensitive?

Reply:
Right, removed logging the values